File: class_exercises/ex2.c
#include <stdio.h>
#include <stdlib.h>
int main(){

    int number[3];
    printf("please enter a number\n");
    
    for (int i =0; i<3; i++){
        scanf("%d", &number[i]);
        printf(number[i]);
    }
    

    return 0;
}

File: class_exercises/ex2.py
def main():
    number = input("please enter numbers to check\n")
    numbers = list(number)
    numbers = [int(x) for x in numbers]
    check_armstrong(numbers, int(number))


def check_armstrong(numbers, number):
    sum = 0
    for i in range(len(numbers)):
        sum = sum + numbers[i]**len(numbers)
    if sum == number:
        print("The number is an armstrong number")
    else:
        print("The number is not armstrong")
    



if __name__ == "__main__":
    main()